documentation/blog/2024-01-01-whats-new-in-svelte-january-2024.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "What's new in Svelte: January 2024"
+description: 'SvelteKit 2 and a much-improved $state rune'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Happy New Year! It's been a busy month for the Svelte maintainers - with tons of new features dropping in the Svelte 5 preview and the [release of SvelteKit 2](https://svelte.dev/blog/sveltekit-2)!
+
+You can find all the new features in both projects below, along with a bunch of resources and sites built with Svelte in the Community Showcase.
+
+Let's jump in...
+
+## What's new in SvelteKit (2.0 and more!)
+
```

Comment:
we just added a new documentation page on performance that might be of interest to people: https://kit.svelte.dev/docs/performance